cyclonedx-bom/src/models/bom.rs

```diff
@@ -284,6 +334,9 @@ impl Validate for Bom {
             results.push(properties.validate_with_context(context)?);
         }
 
+        // todo: validate vulnerabilities, not sure about which approach to take
+        // see different approaches for services vs. properties
```

Comment:
Another TODO. The validation seems to be implemented for vulnerabilities now, any blockers to actually wiring it up?